Tests/docker_native_image_config.json

```diff
@@ -146,7 +146,17 @@
             "ignored_native_images": [
                 "native:8.6"
             ]
+        },
+        {
+            "id": "ThreatConnect v3",
+            "reason": "CIAC-11146",
+            "ignored_native_images": [
+                "native:8.6",
+                "native:dev",
```

Comment:
isnt native dev also 3.11?

Reply:
correct, this is the reason we add it to ignore list

Reply:
mah? its 3.11 in this image and native, so whats the mismatch?